scripts/prepare.js

```diff
@@ -12,23 +18,23 @@ if (!fs.existsSync("analyzers")) {
 }
 
 downloadIfNeeded(
-  "https://repox.sonarsource.com/sonarsource/org/sonarsource/sonarlint/core/sonarlint-language-server/3.5.0.1548/sonarlint-language-server-3.5.0.1548.jar",
+  `https://repox.sonarsource.com/sonarsource/org/sonarsource/sonarlint/core/sonarlint-language-server/${languageServerVersion}/sonarlint-language-server-${languageServerVersion}.jar`,
   "server/sonarlint-ls.jar"
 );
 downloadIfNeeded(
-  "http://repo1.maven.org/maven2/org/sonarsource/javascript/sonar-javascript-plugin/4.0.0.5862/sonar-javascript-plugin-4.0.0.5862.jar",
+  `https://repo1.maven.org/maven2/org/sonarsource/javascript/sonar-javascript-plugin/${sonarJsVersion}/sonar-javascript-plugin-${sonarJsVersion}.jar`,
   "analyzers/sonarjs.jar"
 );
 downloadIfNeeded(
-  "http://repo1.maven.org/maven2/org/sonarsource/php/sonar-php-plugin/2.12.0.2871/sonar-php-plugin-2.12.0.2871.jar",
+  `https://repo1.maven.org/maven2/org/sonarsource/php/sonar-php-plugin/${sonarPhpVersion}/sonar-php-plugin-${sonarPhpVersion}.jar`,
   "analyzers/sonarphp.jar"
 );
 downloadIfNeeded(
-  "http://repo1.maven.org/maven2/org/sonarsource/python/sonar-python-plugin/1.8.0.1496/sonar-python-plugin-1.8.0.1496.jar",
+  `https://repo1.maven.org/maven2/org/sonarsource/python/sonar-python-plugin/${sonarPythonVersion}/sonar-python-plugin-${sonarPythonVersion}.jar`,
   "analyzers/sonarpython.jar"
 );
 downloadIfNeeded(
-  "https://repox.sonarsource.com/sonarsource/org/sonarsource/typescript/sonar-typescript-plugin/1.5.0.2122/sonar-typescript-plugin-1.5.0.2122.jar",
+  `https://repox.sonarsource.com/sonarsource/org/sonarsource/typescript/sonar-typescript-plugin/${sonarTsVersion}/sonar-typescript-plugin-${sonarTsVersion}.jar`,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Split this 161 characters long line (which is greater than 120 authorized). ([javascript:LineLength](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=22&issues=AWNiy4Dc6IBDU_gafCPD&open=AWNiy4Dc6IBDU_gafCPD)

[ISSUE_KEY]: AWNiy4Dc6IBDU_gafCPD